handbook/product/design/design-and-interaction-guidelines.md

```diff
@@ -67,19 +67,18 @@ Drawing the line between what is a code page and what is a web application page
 
 In these cases, we can utilize web application styles inside the search context to give links greater importance through color and provide additional margin and padding to aid readability.
 
+## Accessibility standards
+Our mission is to make it so everyone can code. In order to achieve this it is vital that our applications are designed and built to be accessible to all.
+
+We aim for [WCAG Level AA Conformance](https://www.w3.org/WAI/WCAG2AA-Conformance), use [this reference](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa&technologies=smil%2Cflash%2Csl) to get a better understanding of what this involves.
+
 ### Todo:
 
 * Displaying state in buttons
   * Default to the current state for the icon so that the user understands what is happening now
   * The button text should describe the action that will occur on click
 * Developer friendly as a first principle
   * Keyboard navigation is a priority
-* Accessibility
-  * People of all abilities should have first-class access to code and coding. We cannot fulfill that mission if our application is not accessible
```

Comment:
I liked this sentence "People of all abilities should have first-class access to code and coding" more than "Our mission is to make it so everyone can code" because it is more closely related to what our product enables today (access to code), as opposed to only what our product might enable in the future (access to coding).

I think the high-level goal and design principle is even more important than a WCAG standard, because [WCAG standards can be flawed at times](https://github.com/w3c/wcag/issues/695) too or sometimes it's easy to "nominally" pass the standard without providing the best possible user experience, and for those case it's important to think from first principles.

Reply:
Great point, will preserve the previous text! I agree about the standard, it's not perfect in areas but it's a very good level to aim for 👍